assets/component-mega-menu.css

```diff
@@ -56,6 +56,16 @@
   transition: text-decoration var(--duration-short) ease;
 }
 
+.header--top-center .mega-menu__list {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.header--top-center .mega-menu__list li {
+  width: 16%;
+}
```

Comment:
I feel like maybe the menu items should also be centered. I can see the issue Meli mentioned about this width. If you added a `>` to target the direct child element it should be better.

Reply:
Good catch! Fixed it. I didn't realize it was affecting the L3. I centered it but will let @melissaperreault confirm